include/swift/Parse/Parser.h

```diff
@@ -1555,7 +1567,7 @@ class Parser {
   /// \param inLoc The location of the 'in' keyword, if present.
   ///
   /// \returns true if an error occurred, false otherwise.
-  bool parseClosureSignatureIfPresent(
+  ParserStatus parseClosureSignatureIfPresent(
```

Comment:
We should update the doc comment for `\returns true ...`

Reply:
Done